lib/configure.js

```diff
@@ -456,8 +516,10 @@ module.exports = function() {
       'You\'re ready to auto-deploy using Travis!  Just commit the changes\n' +
       'in .travis.yml and push the commit to the origin/master branch:\n' +
       '\n' +
-      'git commit -m"configure Travis to auto-deploy to GitHub Pages" .travis.yml\n' +
-      'git push origin master\n' +
+      chalk.bold.gray(
```

Comment:
Gray is a little hard to read on my screen for some reason. It is too dark (darker than the screenshot you provided.